topics/contributing/files/example-workflow.ga

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
Why not putting it in `topics/contributing/tutorials/create-new-tutorial/workflows`?

Reply:
because that would have made way too much sense ;) thanks, moved it :)